packages/ui/Scrollable/Scrollable.module.less

```diff
@@ -35,8 +35,4 @@
 		overflow: hidden;
 		flex: 1 1 100%;
 	}
-
-	.contentNative {
```

Comment:
We need to remove the usage of this class from js also.

Reply:
OK